docs/pages/clients/installation.md

```diff
@@ -63,6 +69,14 @@ Make the following changes to allow the Development Client to control project in
 
 You're now ready to start developing your project with the Development Client.
 
+## 4. Optional configuration
+
+### 🍏 iOS
+
+When you start your project on iOS, the metro bundler will be started automatically. This behavior might not be ideal when you want to use `expo start`. Our recommended solution is to remove the `Start Packager` action from building scripts. To do that you need to open the XCode, go to `Project settings` > `Build Phases` and remove the `Start Packager` action.
```

Comment:
I believe yes but want to make sure this is still required with https://github.com/expo/expo-cli/pull/3126